include/bout/region.hxx

```diff
@@ -230,6 +230,33 @@ inline Ind2D operator+(int n, Ind2D rhs) { return rhs += n; }
 inline Ind2D operator-(Ind2D lhs, int n) { return lhs -= n; }
 inline Ind2D operator-(Ind2D lhs, const Ind2D &rhs) { return lhs -= rhs; }
 
+/// Structure to hold various derived "statistics" from a particular region
+struct RegionStats {
+  int numBlocks = 0;           // How many blocks
```

Comment:
These comments could all be doxygen comments

Reply:
Does that mean `///` and on the line before the variable?

Reply:
That's one way, the other way is `///<` on the same line